src/PwdResetEmailSentController.js

```diff
@@ -40,6 +40,17 @@ function (Okta, Enums, FormController, FormType) {
       attributes: { 'data-se': 'pwd-reset-email-sent' },
       formChildren: function () {
         return [
+          FormType.View({
+            View: Okta.View.extend({
+              template: '\
+              <span class="overflow-hidden" style="height:0; display:block" role="alert">{{alert}}</span>\
```

Comment:
can we move the styles to a css class so we can reuse it? Don't we have one already that we use for the security image?
Next question is: is 'alert' good? Or do we want 'status'? I'm not sure about this, that is why I am asking

Last thing: If we need to add this again in another Controller we should definitly move it to a separate View, so we can use the same view wherever we need